Documentation/parameters.par

```diff
@@ -95,6 +95,17 @@ FixTimeStep = 5                      ! Manually chosen maximum time step
 ClusteredLTS = 2                     ! 1 for Global time stepping, 2,3,5,... Local time stepping (advised value 2)
 !ClusteredLTS defines the multi-rate for the time steps of the clusters 2 for Local time stepping
 LtsWeightTypeId = 1                  ! 0=exponential, 1=exponential-balanced, 2=encoded
+
+! Wiggle factor settings:
+! Wiggle factor adjusts time step size by a small factor. This can lead to a slightly better clustering.
+LtsWiggleFactorMin = 1.0 ! Minimal wiggle factor applied to time step size. Should be > 1/rate
```

Comment:
I don't really get it.
I would say CFL_effective = CFL * LtsWiggleFactor
But then why "Should be > 1/rate"